docs/application/native/guides/app-management/app-controls.md

```diff
@@ -808,6 +812,50 @@ The following common application controls are available:
 - [Voice Recorder](common-appcontrols.md#voice-recorder)
 - [VPN Service](common-appcontrols.md#vpn-service)
 
+<a name="URI"></a>
+## Application Control URI
+
+> **Note**
+>
+> Application Control URI is supported since Tizen 5.5.
```

Comment:
```suggestion
> [!Note]
> Application Control URI is supported since Tizen 5.5.
```

Note style has changed, for more information, see https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md#alerts-note-tip-important-caution-warning

Reply:
Done